src/test/scala/cromwell/binding/ThreeStepImportNamespaceSpec.scala

```diff
@@ -0,0 +1,110 @@
+package cromwell.binding
+
+import org.scalatest.{FlatSpec, Matchers}
+
+class ThreeStepImportNamespaceSpec extends FlatSpec with Matchers {
+  val psTaskWdl = """
+    |task ps {
+    |  command {
+    |    ps
+    |  }
+    |  output {
+    |    File procs = stdout()
+    |  }
+    |}""".stripMargin
+
+  val cgrepTaskWdl = """
+    |task cgrep {
+    |  command {
+    |    grep '${pattern}' ${File in_file} | wc -l
+    |  }
+    |  output {
+    |    Int count = read_int(stdout())
+    |  }
+    |}""".stripMargin
+
+  val wcTaskWdl = """
+    |task wc {
+    |  command {
+    |    wc -l ${File in_file}
```

Comment:
I see what you mean.  This might be part of a larger discussion about how lenient engines can/should be in cases like this... doesn't seem too unreasonable to me to say that `read_int` takes the first contiguous non-whitespace and uses that as the integer.